src/test/resources/config/application-testcontainers.yml

```diff
@@ -25,3 +25,22 @@ spring:
         useServerPrepStmts: true
   jpa:
     database-platform: org.hibernate.dialect.MySQL8Dialect
+
+# ===================================================================
+# Application specific properties
+# Add your own application properties here, see the ApplicationProperties class
+# to have type-safe configuration, like in the JHipsterProperties above
+#
+# More documentation is available at:
+# https://www.jhipster.tech/common-application-properties/
+# ===================================================================
+
+application:
+  frontend:
+    recaptcha-project-id:
+    recaptcha-site-key:
+    recaptcha-threshold:
+  recaptcha:
+    project-id:
+    site-key:
+    threshold:
```

Comment:
Do we need to specify these?

Reply:
I believe this is unnecessary now that the `AccountResource` file is updated. Now removed